index.js

```diff
@@ -42,17 +49,17 @@ const expressions = {
     expr: /^\/$/,
     handler: () => BASE_SCC_URL,
   },
+  authReg: {
+    expr: /\/iii\/cas\/login/,
+    handler: (match, parsedURL) => parsedURL.href.replace('catalog.nypl.org', 'ilsstaff.nypl.org'),
+  },
   searchRegWith: {
-    expr: /\/search(~S\d*)?\/([a-zA-Z])?((\w|\W)+)/,
+    expr: /\/search(~S\d*)?\/([a-zA-Z])((\w|\W)+)/,
```

Comment:
Isn't `(\w|\W)` just `(.)`? Again, I could well be missing something...

Reply:
Also you might want to use a non-matching parenthesis for the inner match `(?:\w|\W)`. I know it looks odd and a bit obfuscatory, but you get used to it quickly.